### ISSUE TITLE
Missing sourcemaps for _next/static/chunks/*

### ISSUE THREAD

**Issue by dtbaker** [thredup/rollbar-sourcemap-webpack-plugin#252]
Hello,

As discussed over here: https://github.com/vercel/next.js/discussions/17431

I have connected the app to rollbar and am seeing sourcemaps appear in rollbar, however rollbar is missing sourcemaps for any files under the `/_next/static/chunks/*` path. 

Can you please confirm if this an expected behaviour, or if there's an issue with my setup, or if there might be an issue with which sourcemaps are [chosen for uploading](https://github.com/thredup/rollbar-sourcemap-webpack-plugin/blob/master/src/RollbarSourceMapPlugin.js#L62-L91 )?

**Comment by Ricardonacif** [comment]
I'm still facing this issue, was anyone able to solve it?

**Comment by onggiabayluon** [comment]
Yeah, I think it's because Next.js build files don't have the extension .min.js, and Rollbar cannot distinguish between them, so it chooses to use the sourcemapURL instead.


> The specified file is a source file [is_source_file]
> Rollbar interprets files without a .min.js extension as unminified source files, assuming they don’t require a source map. If your file is minified but doesn’t use the .min.js extension, Rollbar might not process its source map correctly.
https://docs.rollbar.com/docs/source-map-errors#the-specified-file-is-a-source-file-is_source_file


I'm tired of this shit and will move to Sentry instead

**Comment by wrgoto** [comment]
> Yeah, I think it's because Next.js build files don't have the extension .min.js, and Rollbar cannot distinguish between them, so it chooses to use the sourcemapURL instead.
> 
> > The specified file is a source file [is_source_file]
> > Rollbar interprets files without a .min.js extension as unminified source files, assuming they don’t require a source map. If your file is minified but doesn’t use the .min.js extension, Rollbar might not process its source map correctly.
> > https://docs.rollbar.com/docs/source-map-errors#the-specified-file-is-a-source-file-is_source_file
> 
> I'm tired of this shit and will move to Sentry instead

I moved to Sentry a very long time ago lmao.
